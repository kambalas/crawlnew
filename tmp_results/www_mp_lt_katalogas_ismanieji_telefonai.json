[
  {
    "name": "Poco M7 Pro 5G - 8/256GB",
    "price": "209,00 €",
    "error": false
  },
  {
    "name": "Poco F7 Ultra 5G - 12/256GB",
    "price": "709,00 €",
    "error": false
  },
  {
    "name": "Poco F7 Ultra 5G - 16/512GB",
    "price": "769,00 €",
    "error": false
  },
  {
    "name": "Poco F7 Pro 5G - 12/512GB",
    "price": "579,00 €",
    "error": false
  },
  {
    "name": "Poco F7 Pro 5G - 12/256GB",
    "price": "519,00 €",
    "error": false
  },
  {
    "name": "Poco X7 Pro 5G - 8/256GB",
    "price": "309,00 €",
    "error": false
  },
  {
    "name": "Poco X7 Pro 5G - 12/512GB",
    "price": "359,00 €",
    "error": false
  },
  {
    "name": "Poco X7 5G - 12/512GB",
    "price": "279,00 €",
    "error": false
  },
  {
    "name": "Poco X7 5G - 8/256GB",
    "price": "229,00 €",
    "error": false
  },
  {
    "name": "Poco C75 - 8/256GB",
    "price": "119,00 €",
    "error": false
  },
  {
    "name": "Poco X6 Pro 5G - 8/256 GB",
    "price": "283,75 €",
    "error": false
  },
  {
    "name": "Poco M7 Pro 5G - 12/256GB",
    "price": "",
    "error": false
  }
]