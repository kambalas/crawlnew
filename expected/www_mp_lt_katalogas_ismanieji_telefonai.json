[
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Apple iPhone 16e - 512GB",
    "price": "1119.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16e - 256GB",
    "price": "869.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16e - 128GB",
    "price": "729.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro - 128GB",
    "price": "1113.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus - 256GB",
    "price": "1129.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus - 128GB",
    "price": "999.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max - 256GB",
    "price": "1353.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro - 256GB",
    "price": "1233.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 - 128GB",
    "price": "859.00",
    "error": false
  },
  {
    "name": "Apple iPhone 15 - 128 GB",
    "price": "759.00",
    "error": false
  },
  {
    "name": "iPhone 16 Plus 128GB Rausvos spalvos Pink, Mobili prekyba",
    "price": "999.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB, natūralaus titano, 1 nuotrauka",
    "price": "1113.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB, juoda spalva, 1 nuotrauka",
    "price": "869.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 128GB, juoda spalva, 1 nuotrauka",
    "price": "729.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus 128GB, juodas, 1 nuotrauka",
    "price": "999.00",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB, Juodo titano, 1 nuotrauka",
    "price": "1353.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB, juodo titano, 1 nuotrauka",
    "price": "1233.75",
    "error": false
  },
  {
    "name": "Apple iPhone 16, ultramarino mėlynas, 1 nuotrauka",
    "price": "859.00",
    "error": false
  }
]